test/functional/pdo_sqlsrv/pdo_fetch_datetime_time_as_objects.phpt

```diff
@@ -0,0 +1,227 @@
+--TEST--

```

Comment:
Why do these types need to be in a separate test? The tests are very similar...

Reply:
Yes, they are very similar, only the inputs are generated differently, using some different fetch styles. Let me try to combine these two tests.